_rules/first-focusable-elements-are-links-to-content-e53727.md

```diff
@@ -0,0 +1,57 @@
+---
+id: e53727
+name: First focusable elements are links to sections of content
+rule_type: atomic
+description: |
+  This rule checks that the first focusable elements are links referring to elements on the same page
+accessibility_requirements:
+	wcag-technique:G124: # Adding links at the top of the page to each area of the content
+		forConformance: false
+		failed: not satisfied
+		passed: satisfied
+		inapplicable: further testing needed
+input_aspects:
+	- DOM Tree
+	- CSS styling
+authors:
+	- Jean-Yves Moyen
+	- Anne Thyme Nørregard
+---
+
+## Applicability
+
+This rule applies to any [document](#https://www.w3.org/TR/dom/#concept-document) where the [document element](#https://www.w3.org/TR/dom/#document-element) is an HTML `html` element.
+
+## Expectation 1
+
+There is an [initial segment](#initial-segment) of the [focusable](#focusable) elements (in focus order) such that each element in that [intial segment](#initial-segment):

```

Comment:
How does a tester decide where the initial segment ends?
Is that for Expectation 2 to decide? I am not sure I am completely following.